utility/get_ip_addrs.py

```diff
@@ -0,0 +1,57 @@
+""" Look at Ceph storage pods, return ip-addresses """
+import subprocess
+import shlex
+
+
+def get_ip_addrs_for_ceph_type(pod_type, namespace='openshift-storage'):
+    """
+    Get the ip address for all pods of a given type.
+    Args:
+        pod_type (string): pod type (mon, mgr, osd, all for everbody)
+        namespace (string): namespace in -n field of oc commands
+    Returns:
+        List of ip_addresses for all pods of the type specified
+    """
+    ret_list = []
+    look_for = f"rook-ceph-{pod_type}"
+    ipaddrs = get_ip_addrs(namespace)
+    for ceph_pod in ipaddrs:
+        if ceph_pod[0].startswith(look_for):
+            ret_list.append(ceph_pod[1])
```

Comment:
This 'for' block can be translated to list comprehension:
ret_list = [cpod[1] for cpod in ipaddrs if cpod[0].startwith(look_for)]

Reply:
I'll do this.  The change will be slightly different because get_ip_addr will now return a dict.